go/extractor/cli/go-extractor/go-extractor.go

```diff
@@ -115,7 +115,26 @@ func main() {
 	log.Printf("Build flags: '%s'; patterns: '%s'\n", strings.Join(buildFlags, " "), strings.Join(patterns, " "))
 	err := extractor.ExtractWithFlags(buildFlags, patterns)
 	if err != nil {
-		log.Fatalf("Error running go tooling: %s\n", err.Error())
+		diagnosticDir := os.Getenv("CODEQL_EXTRACTOR_GO_DIAGNOSTIC_DIR")
+		errString := err.Error()
+		log.Fatalf("ZZZ: Error running go tooling: %s\nDiagnostic dir is: %s", errString, diagnosticDir)
```

Comment:
Doesn't `log.Fatalf` panic? Also is the ZZZ a temporary addition?

Reply:
Oops, this is temporary code that I pushed by mistake. I'll remove it, as it isn't needed.